kolibri/plugins/setup_wizard/middleware.py

```diff
@@ -29,6 +29,9 @@ def process_request(self, request):
 
         # Don't redirect for URLs that are required for the setup wizard
         allowed_paths = [reverse(name) for name in ALLOWED_PATH_LIST]
+
+        # manually add session bootstrapping api call to allowed path list
+        allowed_paths.append(reverse('session-detail', kwargs={'pk': 'current'}))
```

Comment:
It's doing a `startswith` match, so you could just add 'session' to the `ALLOWED_PATH_LIST` I think?